docs/source/basics/customizing_interface.md

```diff
@@ -111,9 +68,13 @@ The information given here will be rendered in the sidebar of the workflow in th
 * `repository`: A link to the Github repository where the code for the workflow lives,
 * `license`: The license that the workflow code falls under - must be a [SPDX](https://spdx.dev/) identifier.
 
+![Sidebar](../assets/ui/sidebar.png)
+
 ### Customizing Parameter Presentation
 
-When a workflow is registered, each parameter will receive a frontend component to ingest values in the browser. These components will perform HTML-native type validation on inputted values and can be customized from the python code. You can add a parameter to the interface by adding a `LatchParameter` object to your `LatchMetadata` object's parameter dictionary as below:
+When a workflow is registered, each workflow parameter will receive a frontend component to ingest values in the browser. 
+
```

Comment:
I would go into more detail here about how parameters are added and specify that it has to match. Start it off with something like:

> Any input of the main @workflow function can be added & customized on the front end display for ingesting user values in the browser. To add a workflow parameter to the front end simply add a `LatchParameter` object to your `LatchMetadata` object's parameter dictionary:

Reply:
And then add this part below the code snippet: 

> When a workflow is registered, each workflow parameter added to the `LatchMetadata` object will receive a frontend component to ingest values in the browser.